src/main/java/fi/okm/jod/yksilo/controller/ehdotus/MahdollisuudetController.java

```diff
@@ -107,6 +125,13 @@ public List<EhdotusDto> createEhdotus(@RequestBody @Valid LuoEhdotusDto ehdotus)
         .toList();
   }
 
+  // TODO: remove when inference returns type
```

Comment:
The inference endpoint response already contains information about the type, like this:
```
{
  "id": "199611de-c52c-41b3-ae56-ffc8062afc25",
  "koulutus": 0,
  "tyo": 1,
  "score": 0.9896602876413456,  
}
```

Reply:
Implementation changed to use information provided by the response.